src/styles/variables.scss

```diff
@@ -34,6 +38,9 @@ $black: #000 !default;
 $brand: #bd1c2b;
 $highlight: #58cef4;
 
+$color-focus-outline: #51a7e8;
```

Comment:
I prefixed this color variable name with `$color-`. I'm not married to this naming convention (nor do I feel like we need to have all conventions sewn up at this time; we can adapt as we go).

Reply:
We currently have some form fields / buttons using the native focus ring and other inputs using this more Bootstrap-y thing. We need to sort that out. I wonder if we should just go with the native focus ring everywhere. This is fine for the moment.